pkg/network/usm/usm_grpc_monitor_test.go

```diff
@@ -89,6 +89,7 @@ func getGRPCClientsArray(t *testing.T, size int, withTLS bool) ([]*grpc.Client,
 	}
 
 	return res, func() {
+		time.Sleep(time.Second * 2)
```

Comment:
Perhaps a comment as to why it's there could be useful, in case it doesn't get removed in the near future?

Reply:
Good catch!
Regardless of the time it will take to find the race